src/cmd/commit.ts

```diff
@@ -102,48 +152,27 @@ export const getRepo = (values): Record<string, any> => {
   return { remote, branch, email, username, password }
 }
 
-const bootstrapGit = async (values): Promise<void> => {
-  const d = terminal(`cmd:${cmdName}:bootstrapGit`)
-  d.info('Initializing values git repo.')
-  cd(env.ENV_DIR)
-  await $`git init ${env.ENV_DIR}`
-  if (isCli) copyFileSync(`${rootDir}/bin/hooks/pre-commit`, `${env.ENV_DIR}/.git/hooks/pre-commit`)
-  const { remote, branch, email, username, password } = getRepo(values)
-  await $`git config --global --add safe.directory ${env.ENV_DIR}`
-  await $`git config --local user.name ${username}`
-  await $`git config --local user.password ${password}`
-  await $`git config --local user.email ${email}`
-  await $`git checkout -b ${branch}`
-  await $`git remote add origin ${remote}`
-  if (existsSync(`${env.ENV_DIR}/.sops.yaml`)) await nothrow($`git config --local diff.sopsdiffer.textconv "sops -d"`)
-  d.log(`Done bootstrapping git`)
-}
-
 export const commit = async (firstTime = false): Promise<void> => {
   const d = terminal(`cmd:${cmdName}:commit`)
   await validateValues()
   d.info('Preparing values')
   const values = (await hfValues()) as Record<string, any>
-  if (values?._derived?.untrustedCA) {
-    process.env.GIT_SSL_NO_VERIFY = 'true'
-  }
-  if (!existsSync(`${env.ENV_DIR}/.git`)) await bootstrapGit(values)
   const { remote } = getRepo(values)
+  await bootstrapGit(values)
   if (values?.apps!.gitea!.enabled) {
     const { adminPassword } = values.apps!.gitea
     await waitTillAvailable(remote, {
-      // we wait for a 404 as that is the best we can do,
-      // since that is what gitea gives for repos that have nothing public
-      status: 404,
+      status: 200,
       skipSsl: values._derived?.untrustedCA,
       username: 'otomi-admin',
       password: adminPassword,
     })
   }
   await genDrone()
   await encrypt()
-  if (values?.apps?.gitea?.enabled) await commitAndPush(remote)
-  else d.log('The files have been prepared, but you have to commit and push to the remote yourself.')
+  // if (values?.apps?.gitea?.enabled) await commitAndPush()
+  // else d.log('The files have been prepared, but you have to commit and push to the remote yourself.')
+  await commitAndPush()
```

Comment:
So now we `always` push on `otomi commit` CLI call?
If so, please update the command description